### DISCUSSION TITLE
Flag

### DISCUSSION THREAD

**Discussion by cheat2001** [tailwindlabs/heroicons#728] [Ideas]
Please make Cambodia flag my dude.

**Comment by LarsFlieger** [comment]
There is an general `flag` icon... I think if you want to get specific flag icons you have to check another set. Something like this: https://flagicons.lipis.dev/

**Comment by sschoger** [comment]
Hey! No plans to add specific country flag icons. I recommend checking out [Flagpack](https://flagpack.xyz) by Yummy Gum 👍🏻
